app/helpers/application_helper.rb

```diff
@@ -248,6 +248,17 @@ def project_tree(projects, &block)
     Project.project_tree(projects, &block)
   end
 
+  # Returns a lft-sorted project hierarchy only when
```

Comment:
only when what??? OMG, I hate cliffhangers! Please tell me it is a happy end! :wink: